### PR TITLE
Fix weird datetime parsing issue

### PR DESCRIPTION
**Pull request by jbreckman**
`datetime` parsing was broken

### CODE REVIEW

File: paths.pegjs [hunk 1]
```diff
@@ -62,7 +62,7 @@ SimpleExpression = head:WordOrParen tail:(Separator WordOrParen)+ {
   }
 } / EquationPart
  
-DateOperation = operation:("date"/"datetime"/"addmonths"/"addyears"/"adddays"/"addweeks"/"addhours") _ "(" _ path:SimpleExpression secondParameter:(PathSeparator SimpleExpression)? _ ")" {
+DateOperation = operation:("datetime"/"date"/"addmonths"/"addyears"/"adddays"/"addweeks"/"addhours") _ "(" _ path:SimpleExpression secondParameter:(PathSeparator SimpleExpression)? _ ")" {
```
Comment (jbreckman) [inline paths.pegjs:65]: for some reason the order here really mattered ---- it hit "date" and moved on and didn't have a way to back out?

i thought peg would be better about that, but it wasn't.

File: test/date-test.js [hunk 2]
```diff
@@ -125,6 +125,15 @@ t.test('dates',{autoend:true},function(t) {
         });
     });
 
+    t.test('date time math works and nested',{autoend:true},function(t) {
+      return clues(facts,'where.sub(addmonths(adddays(testDate,4),3),datetime("2020-07-05 12:13:15 pm"))=-43995000')
```
Comment (jbreckman) [inline test/date-test.js:129]: 12:13:15 is 43995000 milliseconds after midnight

File: paths.pegjs (hunk 1 above)
Comment (ZJONSSON) [reply paths.pegjs:65]: Yeah peg just takes the first match so that makes sense

**Review by ZJONSSON — APPROVED** [review]
Looks good!!

**Pull request merged**